### DISCUSSION TITLE
Editor shortcuts

### DISCUSSION THREAD

**Discussion by edersong** [enchant97/note-mark#162] [Ideas]
I understood that will not implement WYSIWYG editor, but it could have some editor shortcuts like on memos:
![image](https://github.com/enchant97/note-mark/assets/64137913/42e72745-6937-40eb-b270-e296abb07da2)

but not limited to those, so including for example, code blocks, links, headers, bold, italic etc.

**Comment by enchant97** [comment]
Thanks for the suggestion, I have it already written down on my own personal ideas list! It should be implemented very soon...

**Reply by enchant97** [reply]
@edersong editor toolbar shortcuts now available in V0.10.0

**Reply by edersong** [reply]
Awesome... will test now!
